documentation/TESTING_PLAN.md

```diff
@@ -16,17 +16,28 @@ In general, we're approaching testing bottom-up, with each unit function tested
 
 ### Unit-level tests
 
-- Test that impassible walls are impassible by the player.
-- Test that the level is complete/congratulatory message when and only when all boxes are moved to designated goal cells.
-- Test when a level is complete, the user is prompted to move to the next level or jump to a different level.
-- Test that all boxes are pushable by the player.
-- Test the score is tracking player movement.
-- Test that the undo button undoes moves.
-- Test the reset button reset’s the board properly.
-- Test that the 'Sart' button initiates the default game by dropping the player into level 1.
-- Test that the drop-down menu is clickable by the player and allows them to jump to different levels.
-- Test that the directional keys move the character in the proper direction up, down, left, and right.
+- undo() should load the previous game state.
+- reset() should the current level to its original state.
+- move(U) should allow player to move up.
```

Comment:
Should we specify here what is allowed for the player to move up. If a player moves up only when it's a goal or an empty cell. but if there's a wall, box-wall, box-box, etc, nothing should happen.

Reply:
"previous game state" might not be entirely clear to a reader even if it's accurate -- maybe "undo the last move" makes more sense since that's what the user experiences. Game state is somewhat of an implementation detail.